### ISSUE TITLE
Avatar: fallback svg doesn't have accessible text

### ISSUE THREAD

**Issue by baumandm** [chakra-ui/chakra-ui#3455]
# 🐛 Bug report

The `<Avatar>` component displays a fallback svg when no `src` or `name` attribute is provided. 

This fails an [Axe](https://github.com/dequelabs/axe-core) accessibility check:
```
serious: svg elements with an img role have an alternative text https://dequeuniversity.com/rules/axe/4.1/svg-img-alt?application=axeAPI
```

Manually adding an `aria-label` attribute places it on the wrapper `<span`> rather than on the `<svg>` tag itself.

## 💥 Steps to reproduce

```<Avatar />```

or

```<Avatar aria-label="avatar" />```

## 💻 Link to reproduction

https://codesandbox.io/s/proud-voice-pzbqs?file=/src/App.tsx

## 🧐 Expected behavior

Details on passing the Axe rule can be found [here](https://dequeuniversity.com/rules/axe/4.1/svg-img-alt).

IMHO either a default `aria-label` attribute should be provided in this scenario, or a manually-provided `aria-label` should be applied to the `svg` rather than the `span`.  Alternately a `<title>` tag could be added to the fallback svg itself.

**Comment by segunadebayo** [comment]
Pushed a fix for this. Thanks @baumandm
